packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-editor.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+	<div class="value-editor">
+		<div>
```

Comment:
For divs that hold larger sections/containers I'd suggest to use `section`. eg. the parent div and the children just after that.